fred2/sexp_tree.cpp

```diff
@@ -2272,7 +2274,8 @@ int sexp_tree::get_default_value(sexp_list_item *item, char *text_buf, int op, i
 	h = item_handle;
 	index = item_index;
 	type = query_operator_argument_type(op, i);
-	switch (type) {
+	switch (type)
+	{
```

Comment:
There are only a few whitespace changes compared to the rest of the diff.  I made the changes so that those lines would match the rest of the function.  If you look at the function as a whole, it uses inconsistent formatting before the diff but consistent formatting after.